libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/ScheduledMailDialogRenderer/ScheduledMailDialogRenderer.tsx

```diff
@@ -388,9 +394,12 @@ export class ScheduledMailDialogRendererUI extends React.PureComponent<
                     }
                     return recipient;
                 });
-                this.setState({
+                const newState = {
+                    ...this.state,
                     selectedRecipients: processedRecipients,
-                });
+                };
+                this.originalEditState = newState;
```

Comment:
the original state needs to be saved once the email addresses have been resolved/converted to workspace users